src/Kmd.Logic.CitizenDocuments.Client/Kmd.Logic.CitizenDocuments.Client.csproj

```diff
@@ -8,6 +8,10 @@
 
   <ItemGroup>
     <PackageReference Include="Kmd.Logic.Identity.Authorization" Version="1.0.0" />
+    <PackageReference Include="StyleCop.Analyzers" Version="1.1.118">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
```

Comment:
Shouldn't this be inherited from the `Directory.Build.props` file?

Reply:
Was not sure before, but now removed from citizen document sample and client project. 👍